inst/shiny/tabs/nca.R

```diff
@@ -18,8 +18,19 @@ observeEvent(data(), {
   updateSelectInput(
     session,
     inputId = "select_analyte",
-    label = "Choose the analyte :",
-    choices = unique(data()$ANALYTE)
+    label = "Choose the Analyte(s) :",
+    choices = unique(data()$ANALYTE),
+    selected = unique(data()$ANALYTE)[1]
+  )
+})
+
+observeEvent(data(), {
+  updateSelectInput(
+    session,
+    inputId = "select_pcspec",
+    label = "Choose the Specimen Type(s) :",
+    choices = unique(data()$PCSPEC),
+    selected = unique(data()$PCSPEC)[1]
```

Comment:
nitpick(non-blocking): No need to specify selected argument, upon choices change this will automatically change to the first element.
```suggestion
    choices = unique(data()$PCSPEC)
```